crates/ra_ide_api/src/lib.rs

```diff
@@ -243,10 +244,34 @@ impl<T> RangeInfo<T> {
 
 #[derive(Debug)]
 pub struct CallInfo {
-    pub label: String,
+    pub signature: FunctionSignature,
+    pub active_parameter: Option<usize>,
+}
+
+/// Contains information about a function signature
+#[derive(Debug)]
+pub struct FunctionSignature {
```

Comment:
I think this could be moved to display module: this is strictly a dispaly sig, for analysis one should use HIR